docs/source/membership/membership.md

```diff
@@ -7,7 +7,7 @@ Note: this topic describes a network that does not use a "system channel", a cha
 Because Fabric is a permissioned network, blockchain participants need a way to prove their identity to the rest of the network in order to transact on the network. If you've read through the documentation on [Identity](../identity/identity.html)
 you've seen how a Public Key Infrastructure (PKI) can provide verifiable identities through a chain of trust. How is that chain of trust used by the blockchain network?
 
-Certificate Authorities issue identities by generating a public and private key which forms a key-pair that can be used to prove identity. Because a private key can never be shared publicly, a mechanism is required to enable that proof which is where the MSP comes in. For example, a peer uses its private key to digitally sign, or endorse, a transaction.  The MSP on the ordering service contains the peer's public key which is then used to verify that the signature attached to the transaction is valid. The private key is used to produce a signature on a transaction that only the corresponding public key, that is part of an MSP, can match. Thus, the MSP is the mechanism that allows that identity to be trusted and recognized by the rest of the network without ever revealing the member’s private key.
```

Comment:
> The MSP on the ordering service contains the peer's public key which is then used to verify that the signature attached to the transaction is valid

This is completely and utterly false :) 

> The private key is used to produce a signature on a transaction that only the corresponding public key, that is part of an MSP, can match

While this may be true empirically, it has absolutely no importance as that's not a security requirement.
Instead, what we need is that no one can forge a signature on a message for a specific public key, but not that there can not be another public key that the signature verifies under. For instance, if there is a scheme with a "bad" public key that any string is a valid signature under that public key, the general signature scheme may still be safe if that "bad" public key is avoided with overwhelming probability.